cml_tools/gen_guestos.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+OSTMPL=$1
+OSNAME=$2
+ROOTFS_TARBALL="$3"
+TRUSTME_VERSION="1"
+
+SCRIPT_DIR="$4"
+DEPLOY_DIR_IMAGE="$7/out"
+TRUSTME_HARDWARE="x86"
+
+
+CFG_OVERLAY_DIR="${SCRIPT_DIR}/config_overlay"
+CONFIG_CREATOR_DIR="${SCRIPT_DIR}/config_creator"
+PROTO_FILE_DIR="$5"
+PROVISIONING_DIR="${SCRIPT_DIR}/device_provisioning"
+ENROLLMENT_DIR="${PROVISIONING_DIR}/oss_enrollment"
+TEST_CERT_DIR="$6"
```

Comment:
plz update to match paths by debian package (remove SCRIPT_DIR) parameters

Validate parameters